fee-abstraction/services/scaffolder/fee.go

```diff
@@ -0,0 +1,101 @@
+package scaffolder
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/blang/semver/v4"
+	"github.com/ignite/cli/v28/ignite/pkg/errors"
+	"github.com/ignite/cli/v28/ignite/pkg/placeholder"
+	"github.com/ignite/cli/v28/ignite/pkg/xgenny"
+
+	"github.com/ignite/apps/fee-abstraction/pkg/xgit"
+	"github.com/ignite/apps/fee-abstraction/template"
+)
+
+const (
+	feeAbsRepo = "github.com/osmosis-labs/fee-abstraction"
+)
+
+var versionMap = map[uint64]string{
```

Comment:
I don't think this is useful as the template uses v8.

Reply:
I kept the version flag because we can use different v8 versions, but I removed the v7 version from the map

Reply:
What do you mean by different 8 versions?

Reply:
diferent minor versions like `v8.0.2`, `v8.0.1`....

Reply:
Right but different minor version won't change the package name. It changes only on major. I don't think we need that map actually